examples/supp_plugin/plugin/plugin_static.rs

```diff
@@ -0,0 +1,27 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#[no_mangle]
+pub static mut plugin_method: PluginMethod = PluginMethod {
```

Comment:
What is the intention to make this data structure mutable? It seems that the [tee_supplicant logic](https://github.com/OP-TEE/optee_client/blob/7c9c423d00e96bf51debd5fe10fd70dce83be5cc/tee-supplicant/src/plugin.c#L85) only needs a constant symbol to look up.

Reply:
After removing the `mut`, this error occurs: 
```
error[E0277]: `*mut u8` cannot be shared between threads safely.
  = help: within `optee_teec::extension::PluginMethod`, the trait `std::marker::Sync` is not implemented for `*mut u8`
   = note: required because it appears within the type `optee_teec::extension::PluginMethod`
   = note: shared static variables must have a type that implements `Sync`
```